modules/express/src/clientRoutes.ts

```diff
@@ -1609,12 +1609,7 @@ export function setupAPIRoutes(app: express.Application, config: Config): void {
   // API v2
 
   // create keychain
-  app.post(
-    '/api/v2/:coin/keychain/local',
-    parseBody,
-    prepareBitGo(config),
-    promiseWrapper(handleV2CreateLocalKeyChain)
-  );
+  router.post('express.keychain.local', [prepareBitGo(config), typedPromiseWrapper(handleV2CreateLocalKeyChain)]);
```

Comment:
Even though the function already satisfies the type, can we update `handleV2CreateLocalKeyChain` to follow the same standard we have for other typed routes? (i.e. update the req type)